website/content/en/highlights/2021-12-14-improved-concurrency-model.md

```diff
@@ -0,0 +1,34 @@
+---
+date: "2021-12-14"
+title: "Improved concurrency model"
+description: "New parallel processing with improved performance"
+authors: ["barieom"]
+pr_numbers: [10265]
+release: "0.19.0"
+hide_on_release_notes: false
+badges:
+  type: announcement
+---
+
+We've released an improved concurrency model that provides measurable performance
+improvements and enables more efficient vertical scaling.
+
+Previously, Vector set the limit to active tasks to the number of available CPUs
+in a given environment. Transforms were run sequentially, which made transforms
+a bottleneck in Vector pipelines.
+
+With this new release, Vector will perform faster when a transform is a bottleneck,
+assuming that more CPUs are available to share their work. This works by Vector
+determine whether an individual transform will be processed in parallel when there
+is a sufficient load to the environment.
+
+To expand, this improvement works by spinning up multiple short-lived tasks
+that concurrently run the same transform logic on separate batches of events.
+In other words, they perform the processing for chunks of events at a time and
+allowing a certain number of those tasks to be active at a time.
```

Comment:
```suggestion
This improvement works by spinning up multiple short-lived tasks
that concurrently run the same transform logic on separate batches of events.
```

I think those two sentences were saying the same thing. You could maybe move this sentence up into the above paragraph now rather than having it stand alone.